spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseDeleteOrphanFilesSparkAction.java

```diff
@@ -205,15 +213,16 @@ private String jobDesc() {
     JavaRDD<String> subDirRDD = sparkContext().parallelize(subDirs, parallelism);
 
     Broadcast<SerializableConfiguration> conf = sparkContext().broadcast(hadoopConf);
-    JavaRDD<String> matchingLeafFileRDD = subDirRDD.mapPartitions(listDirsRecursively(conf, olderThanTimestamp));
+    JavaRDD<String> matchingLeafFileRDD =
```

Comment:
nit: an alternative way of writing this (with 4 spaces for continued indentation)

```
JavaRDD<String> matchingLeafFileRDD = subDirRDD.mapPartitions(
    listDirsRecursively(conf, olderThanTimestamp, pathFilter)
);
```